auto_ria/auto_ria/spiders/ria_spider.py

```diff
@@ -9,25 +10,41 @@ class RiaSpider(scrapy.Spider):
     start_urls = []
     page_counter = 1
 
-    def __init__(self, url=None, *args, **kwargs):
+    def __init__(self, url=None, f=None, *args, **kwargs):
```

Comment:
`f` is too short for a variable name, it is better to use full name `input_file` or something like that.
Also it is not clear if `f` is a file object of file name, it would be good to have a docstring or a type hint.

Reply:
`f` is command line argument here, I used so short name by purpose... I'll see if there's a better way to handle command line parameters, but not sure.
That's filename, I'll add a docstring and typehint.